### ISSUE TITLE
Kubeadm pull images failed

### ISSUE THREAD

**Issue by nobiit** [kubernetes/kubernetes#110077]
### What happened?

I can't initialize Kubernetes 1.24.0 cluster, failed at `kubeadm config images pull` step

```
+ sudo kubeadm config images pull
failed to pull image "k8s.gcr.io/kube-apiserver:v1.24.0": output: time="2022-05-17T02:10:44+07:00" level=fatal msg="pulling image: rpc error: code = Unimplemented desc = unknown service runtime.v1alpha2.ImageService"
, error: exit status 1
To see the stack trace of this error execute with --v=5 or higher
```

### What did you expect to happen?

I can initialize Kubernetes 1.24.0 cluster without error

### How can we reproduce it (as minimally and precisely as possible)?

1. Prepare a computer with docker/containerd pre-installed and make sure it works properly
2. Install kubeadm/kubelet/kubectl 1.24.0 using apt
3. Run `kubeadm config images pull`

### Anything else we need to know?

- I opened an issue at kubeadm, and it looks like it's on k8s image maintenance.
https://github.com/kubernetes/kubeadm/issues/2696
- Everything works with 1.23.6, fails on 1.24.0

### Kubernetes version

WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.0", GitCommit:"4ce5a8954017644c5420bae81d72b09b735c21f0", GitTreeState:"clean", BuildDate:"2022-05-03T13:46:05Z", GoVersion:"go1.18.1", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.4
The connection to the server localhost:8080 was refused - did you specify the right host or port?

### Cloud provider

Ubuntu

### OS version

Ubuntu 20.04

### Install tools

apt

### Container runtime (CRI) and version (if applicable)

containerd

**Comment by neolit123** [comment]
@nobiit also reports that ctr fails to extract 1.24 image layers.

https://github.com/kubernetes/kubeadm/issues/2696#issuecomment-1128217146

**Comment by neolit123** [comment]
As noted on the issue this seems like either a problem with containerd or the images. Not s kubeadm issue, per se.

**Comment by neolit123** [comment]
@nobiit from the images that kubeadm lists, does this only happen for the kube-apiserver image?

**Comment by mk46** [comment]
> As noted on the issue this seems like either a problem with containerd or the images. Not a kubeadm issue, per se.

Issue is related to containerd, I try to reproduce the same. It's working with fixes provided by @dims in https://lifesaver.codes/answer/failed-to-create-kubelet-unknown-service-runtime-v1alpha2-runtimeservice-73189
`containerd config default > /etc/containerd/config.toml`

**Comment by neolit123** [comment]
/remove-sig release
/sig node
(But the issue seems external)

**Comment by mk46** [comment]
> @mk46 do you know if there is a logged issue in the containerd repository?

Yes but not exactly for v1.24. See  https://github.com/containerd/containerd/issues/4581 and https://github.com/kubernetes-sigs/cri-tools/issues/710

**Comment by mk46** [comment]
`disabled_plugins = ["cri"]`  in /etc/containerd/config.toml is actual cause. this is default config when installing containerd with apt.
Possible solution may be

1. Commenting disabled_plugins = ["cri"]  or
2. Removing /etc/containerd/config.toml or
3. containerd config default > /etc/containerd/config.toml 

and then restarting containerd service.
